.setup/data/courses/sample.yml

```diff
@@ -255,10 +255,75 @@ gradeables:
   g_grade_start_date: 9997-12-31 23:59:59
   eg_team_assignment: true
   eg_max_team_size: 3
-  eg_max_random_submissions: 0
```

Comment:
Why do we need to update 'open_team_homework' config?

Reply:
This change shouldn't be in this PR, nice catch. I'm currently working on sample submission for team assignments, and I made some changes to the yml file. I'll undo this line from this PR